### PR TITLE
New version: WHONET.AMRIE version 24.2.16

### PR DESCRIPTION
**Pull request by AClark-WHONET**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/140216)

### CODE REVIEW

**Comment by AClark-WHONET** [comment]
@microsoft-github-policy-service agree company="Brigham and Women's Hospital"

**Comment by drshtmstry** [comment]
Duplicate of #139824

**Comment by drshtmstry** [comment]
@AClark-WHONET Added PR https://github.com/AClark-WHONET/AMRIE/pull/5 which also supports formatted GitHub release notes.

File: manifests/w/WHONET/AMRIE/24.2.16/WHONET.AMRIE.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: WHONET.AMRIE
+PackageVersion: 24.2.16
+InstallerLocale: en-US
+InstallerType: wix
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.DotNet.DesktopRuntime.6
```
[the remark this answers is not in the document]
Comment (AClark-WHONET) [reply manifests/w/WHONET/AMRIE/24.2.16/WHONET.AMRIE.installer.yaml:10]: Thanks @drshtmstry, do you know where this value should be updated?

[the remark this answers is not in the document]
Comment (drshtmstry) [reply manifests/w/WHONET/AMRIE/24.2.16/WHONET.AMRIE.installer.yaml:10]: I have already submitted PR for the 24.2.16 version [here](https://github.com/microsoft/winget-pkgs/pull/139824). Which is pending approval from moderators.

[the remark this answers is not in the document]
Comment (AClark-WHONET) [reply manifests/w/WHONET/AMRIE/24.2.16/WHONET.AMRIE.installer.yaml:10]: Thanks. If we can get this to work without including the additional project from vedantmgoyal2009 that would be preferable. I submitted a few changes to the publish-to-winget.yml file on my repository which resolved the inability to get the target version, etc. If it works without the additional project, then I will cancel the associated pull request from you which would merge that in.

**Comment by stephengillie** [comment]
Hi @AClark-WHONET,

The package didn't pass a Defender or similar security scan. This might be a false positive and we can rescan tomorrow.

(Automated response - build 771.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Installation failed with exit code -1978335187 2024-02-20 08:18:03.897 [CLI ] Installer [Arm64,burn,Unknown,] not applicable: Machine is not compatible with Arm64 2024-02-20 08:18:53.251 [FAIL] Installer failed security check. Url: https://github.com/AClark-WHONET/AMRIE/releases/download/v24.2.16/AMR_Interpretation_Engine_v24.2.16.msi Result: 0x80004005 2024-02-20 08:18:53.251 [CLI ] Terminating context: 0x8a15002d at D:\a\_work\1\s\external\pkg\src\AppInstallerCLICore\Workflows\DownloadFlow.cpp:1e6 MSI (s) (BC:D8) [08:18:16:384]: Executing op: ComponentRegister(ComponentId={5FF1FFB8-9BF6-5BBF-A666-98C9128ABCC6},KeyPath=22*85*\SOFTWARE\Microsoft\Windows\Windows Error Reporting\RuntimeExceptionHelperModules\C:\Program Files\dotnet\shared\Microsoft.NETCore.App\6.0.27\mscordaccore.dll,State=3,,Disk=1,SharedDllRefCount=0,BinaryType=1) MSI (s) (BC:D8) [08:18:17:962]: Executing op: RegOpenKey(Root=-2147483646,Key=SOFTWARE\Microsoft\Windows\Windows Error Reporting\RuntimeExceptionHelperModules,,BinaryType=1,,) Property(S): ErrorDialog = ErrorDlg MSI (s) (BC:D8) [08:18:26:275]: Windows Installer installed the product. Product Name: Microsoft .NET Runtime - 6.0.27 (x64). Product Version: 48.108.8828. Product Language: 1033. Manufacturer: Microsoft Corporation. Installation success or error status: 0. MSI (s) (BC:40) [08:18:29:150]: Windows Installer installed the product. Product Name: Microsoft .NET Host FX Resolver - 6.0.27 (x64). Product Version: 48.108.8828. Product Language: 1033. Manufacturer: Microsoft Corporation. Installation success or error status: 0. MSI (s) (BC:A8) [08:18:29:978]: Windows Installer installed the product. Product Name: Microsoft .NET Host - 6.0.27 (x64). Product Version: 48.108.8828. Product Language: 1033. Manufacturer: Microsoft Corporation. Installation success or error status: 0. MSI (s) (BC:84) [08:18:34:822]: Windows Installer installed the product. Product Name: Microsoft Windows Desktop Runtime - 6.0.27 (x64). Product Version: 48.108.8836. Product Language: 1033. Manufacturer: Microsoft Corporation. Installation success or error status: 0. 2024-02-20 08:17:33.262 [CLI ] Installer [Arm64,burn,Unknown,] not applicable: Machine is not compatible with Arm64 2024-02-20 08:17:33.262 [CLI ] Installer [X64,burn,Unknown,] not applicable: Machine is not compatible with X64 2024-02-20 08:17:57.481 [CLI ] Installer [X64,wix,Unknown,en-US] not applicable: Machine is not compatible with X64 2024-02-20 08:18:05.154 [FAIL] Installer failed security check. Url: https://github.com/AClark-WHONET/AMRIE/releases/download/v24.2.16/AMR_Interpretation_Engine_v24.2.16.msi Result: 0x80004005 2024-02-20 08:18:05.154 [CLI ] Terminating context: 0x8a15002d at D:\a\_work\1\s\external\pkg\src\AppInstallerCLICore\Workflows\DownloadFlow.cpp:1e6 MSI (s) (48:30) [08:17:43:537]: Executing op: ComponentRegister(ComponentId={C522FD37-C3F6-5BCD-9834-1A7EF0A72DE7},KeyPath=02*85*\SOFTWARE\Microsoft\Windows\Windows Error Reporting\RuntimeExceptionHelperModules\C:\Program Files\dotnet\shared\Microsoft.NETCore.App\6.0.27\mscordaccore.dll,State=3,,Disk=1,SharedDllRefCount=0,BinaryType=0) MSI (s) (48:30) [08:17:45:022]: Executing op: RegOpenKey(Root=-2147483646,Key=SOFTWARE\Microsoft\Windows\Windows Error Reporting\RuntimeExceptionHelperModules,,BinaryType=0,,) MSI (s) (48:30) [08:17:45:287]: Windows Installer installed the product. Product Name: Microsoft .NET Runtime - 6.0.27 (x86). Product Version: 48.108.8828. Product Language: 1033. Manufacturer: Microsoft Corporation. Installation success or error status: 0. MSI (s) (48:A4) [08:17:45:850]: Windows Installer installed the product. Product Name: Microsoft .NET Host FX Resolver - 6.0.27 (x86). Product Version: 48.108.8828. Product Language: 1033. Manufacturer: Microsoft Corporation. Installation success or error status: 0. MSI (s) (48:C4) [08:17:46:522]: Windows Installer installed the product. Product Name: Microsoft .NET Host - 6.0.27 (x86). Product Version: 48.108.8828. Product Language: 1033. Manufacturer: Microsoft Corporation. Installation success or error status: 0. MSI (s) (48:74) [08:17:51:365]: Windows Installer installed the product. Product Name: Microsoft Windows Desktop Runtime - 6.0.27 (x86). Product Version: 48.108.8836. Product Language: 1033. Manufacturer: Microsoft Corporation. Installation success or error status: 0.

(Automated response - build 771.)

**Comment by drshtmstry** [comment]
@stephengillie This is duplicate PR. Original is here https://github.com/microsoft/winget-pkgs/pull/139824

**Comment by stephengillie** [comment]
Sorry for the delay with the other one.

**Comment by stephengillie** [comment]
Close with reason: Duplicate of #139824;

**Pull request closed without merging**